### PR TITLE
Fix UI Tests

### LINKED ISSUE #154: Main fails UI tests
On `main` currently, `yarn run oss:test` fails (see the [README](https://github.com/openbao/openbao/blob/main/ui/README.md#running-tests)). This is likely as a result of the plugin removal from #64 and various associated PRs. 

Sample output below:

```bash
$ yarn run test:oss
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating

$ bao server -dev -dev-ha -dev-transactional -dev-root-token-id=root -dev-listen-address=127.0.0.1:9200
$ ember test -f=!enterprise
==> OpenBao server configuration:

Administrative Namespace: 
             Api Address: http://127.0.0.1:9200
                     Cgo: disabled
         Cluster Address: https://127.0.0.1:9201
   Environment Variables: BERRY_BIN_FOLDER, COLORTERM, DBUS_SESSION_BUS_ADDRESS, DEBUGINFOD_URLS, DESKTOP_SESSION, DISPLAY, DOCKER_HOST, EDITOR, GDMSESSION, GDM_LANG, GIT_EDITOR, GNOME_SETUP_DISPLAY, GNOME_TERMINAL_SCREEN, GNOME_TERMINAL_SERVICE, HISTCONTROL, HISTIGNORE, HISTSIZE, HOME, HOSTNAME, INIT_CWD, LANG, LESSOPEN, LOGNAME, LS_COLORS, MAIL, MAKEFLAGS, MAKELEVEL, MAKE_TERMERR, MAKE_TERMOUT, MEMORY_PRESSURE_WATCH, MEMORY_PRESSURE_WRITE, MFLAGS, OLDPWD, PATH, PROJECT_CWD, PWD, QT_IM_MODULE, SESSION_MANAGER, SHELL, SHLVL, SSH_AUTH_SOCK, SYSTEMD_EXEC_PID, TERM, USER, USERNAME, VIRT_ADDR, VISUAL, VTE_VERSION, WAYLAND_DISPLAY, XAUTHORITY, XDG_CURRENT_DESKTOP, XDG_DATA_DIRS, XDG_MENU_PREFIX, XDG_RUNTIME_DIR, XDG_SESSION_CLASS, XDG_SESSION_DESKTOP, XDG_SESSION_TYPE, XMODIFIERS, YARN_IGNORE_CWD, _, npm_config_user_agent, npm_execpath, npm_lifecycle_event, npm_node_execpath, npm_package_json, npm_package_name, npm_package_version
              Go Version: go1.22.1
              Listener 1: tcp (addr: "127.0.0.1:9200", cluster address: "127.0.0.1:9201", max_request_duration: "1m30s", max_request_size: "33554432", tls: "disabled")
               Log Level: 
                   Mlock: supported: true, enabled: false
           Recovery Mode: false
                 Storage: inmem_transactional_ha (HA available)
                 Version: OpenBao v1.14.9, built 2024-03-06T22:44:51Z
             Version Sha: c2d48fcbccafd5fbc3f530c6440d1ad74c49a32f

==> OpenBao server started! Log data will stream in below:

2024-03-07T11:10:12.064-0500 [INFO]  proxy environment: http_proxy="" https_proxy="" no_proxy=""
2024-03-07T11:10:12.064-0500 [WARN]  no `api_addr` value specified in config or in BAO_API_ADDR; falling back to detection if possible, but this value should be manually set
2024-03-07T11:10:12.065-0500 [INFO]  core: Initializing version history cache for core
2024-03-07T11:10:12.066-0500 [INFO]  core: security barrier not initialized
2024-03-07T11:10:12.066-0500 [INFO]  core: security barrier initialized: stored=1 shares=1 threshold=1
2024-03-07T11:10:12.078-0500 [INFO]  core: post-unseal setup starting
2024-03-07T11:10:12.079-0500 [INFO]  core: loaded wrapping token key
2024-03-07T11:10:12.079-0500 [INFO]  core: successfully setup plugin catalog: plugin-directory=""
2024-03-07T11:10:12.079-0500 [INFO]  core: no mounts; adding default mount table
2024-03-07T11:10:12.080-0500 [INFO]  core: successfully mounted: type=cubbyhole version="v1.14.9+builtin.bao" path=cubbyhole/ namespace="ID: root. Path: "
2024-03-07T11:10:12.081-0500 [INFO]  core: successfully mounted: type=system version="v1.14.9+builtin.bao" path=sys/ namespace="ID: root. Path: "
2024-03-07T11:10:12.082-0500 [INFO]  core: successfully mounted: type=identity version="v1.14.9+builtin.bao" path=identity/ namespace="ID: root. Path: "
2024-03-07T11:10:12.083-0500 [INFO]  core: successfully mounted: type=token version="v1.14.9+builtin.bao" path=token/ namespace="ID: root. Path: "
2024-03-07T11:10:12.083-0500 [INFO]  rollback: Starting the rollback manager with 256 workers
2024-03-07T11:10:12.084-0500 [INFO]  rollback: starting rollback manager
2024-03-07T11:10:12.084-0500 [INFO]  core: restoring leases
2024-03-07T11:10:12.085-0500 [INFO]  expiration: lease restore complete
2024-03-07T11:10:12.085-0500 [INFO]  identity: entities restored
2024-03-07T11:10:12.085-0500 [INFO]  identity: groups restored
2024-03-07T11:10:12.086-0500 [INFO]  core: Recorded vault version: vault version=1.14.9 upgrade time="2024-03-07 16:10:12.08600982 +0000 UTC" build date=2024-03-06T22:44:51Z
2024-03-07T11:10:12.086-0500 [INFO]  core: post-unseal setup complete
2024-03-07T11:10:12.086-0500 [INFO]  core: root token generated
2024-03-07T11:10:12.086-0500 [INFO]  core: pre-seal teardown starting
2024-03-07T11:10:12.086-0500 [INFO]  rollback: stopping rollback manager
2024-03-07T11:10:12.087-0500 [INFO]  core: pre-seal teardown complete
2024-03-07T11:10:12.087-0500 [INFO]  core.cluster-listener.tcp: starting listener: listener_address=127.0.0.1:9201
2024-03-07T11:10:12.087-0500 [INFO]  core.cluster-listener: serving cluster requests: cluster_listen_address=127.0.0.1:9201
2024-03-07T11:10:12.087-0500 [INFO]  core: vault is unsealed
2024-03-07T11:10:12.087-0500 [INFO]  core: entering standby mode
2024-03-07T11:10:12.087-0500 [INFO]  core: acquired lock, enabling active operation
2024-03-07T11:10:12.091-0500 [INFO]  core: post-unseal setup starting
2024-03-07T11:10:12.091-0500 [INFO]  core: loaded wrapping token key
2024-03-07T11:10:12.091-0500 [INFO]  core: successfully setup plugin catalog: plugin-directory=""
2024-03-07T11:10:12.092-0500 [INFO]  core: successfully mounted: type=system version="v1.14.9+builtin.bao" path=sys/ namespace="ID: root. Path: "
2024-03-07T11:10:12.093-0500 [INFO]  core: successfully mounted: type=identity version="v1.14.9+builtin.bao" path=identity/ namespace="ID: root. Path: "
2024-03-07T11:10:12.093-0500 [INFO]  core: successfully mounted: type=cubbyhole version="v1.14.9+builtin.bao" path=cubbyhole/ namespace="ID: root. Path: "
2024-03-07T11:10:12.094-0500 [INFO]  core: successfully mounted: type=token version="v1.14.9+builtin.bao" path=token/ namespace="ID: root. Path: "
2024-03-07T11:10:12.094-0500 [INFO]  rollback: Starting the rollback manager with 256 workers
2024-03-07T11:10:12.094-0500 [INFO]  rollback: starting rollback manager
2024-03-07T11:10:12.094-0500 [INFO]  core: restoring leases
2024-03-07T11:10:12.094-0500 [INFO]  identity: entities restored
2024-03-07T11:10:12.094-0500 [INFO]  identity: groups restored
2024-03-07T11:10:12.094-0500 [INFO]  expiration: lease restore complete
2024-03-07T11:10:12.095-0500 [INFO]  core: post-unseal setup complete
2024-03-07T11:10:13.089-0500 [INFO]  expiration: revoked lease: lease_id=auth/token/root/h1b7f9f6a226885d8f05fd3d3b54cd1d272bc8aef0fce740bb932843d783b18b5
2024-03-07T11:10:13.093-0500 [INFO]  core: successful mount: namespace="" path=secret/ type=kv version=""
2024-03-07T11:10:13.103-0500 [INFO]  secrets.kv.kv_5620ae38: collecting keys to upgrade
2024-03-07T11:10:13.103-0500 [INFO]  secrets.kv.kv_5620ae38: done collecting keys: num_keys=1
2024-03-07T11:10:13.103-0500 [INFO]  secrets.kv.kv_5620ae38: upgrading keys finished
WARNING! dev mode is enabled! In this mode, Vault runs entirely in-memory
and starts unsealed with a single unseal key. The root token is already
authenticated to the CLI, so you can immediately begin using Vault.

You may need to set the following environment variables:

    $ export BAO_ADDR='http://127.0.0.1:9200'

The unseal key and root token are displayed below in case you want to
seal/unseal the Vault or re-authenticate.

Unseal Key: Dh5/qldOPjB0EUNgtn17WYHAns3dwjz7ey/9dXpkUu4=
Root Token: root

Development mode should NOT be used in production installations!

VAULT SERVER READY
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
WARNING: Node v20.10.0 is not tested against Ember CLI on your platform. We recommend that you use the most-recent "Active LTS" version of Node.js. See https://git.io/v7S5n for details.
Environment: test
⠇ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠏ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠋ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠙ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠹ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠦ building... [Babel: @ember/test-helpers > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠧ building... [Babel: ember-qunit > applyPatches]Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
⠹ building... [Babel: ember-modifier > applyPatches]unexpectedly found "{{! template-lint-d" when slicing source, but expected "@destinationElement"
unexpectedly found "{{! template-l" when slicing source, but expected "@renderInPlace"
⠋ building... [Babel: @hashicorp/design-system-components > applyPatches]unexpectedly found "{{#if this.showConf" when slicing source, but expected "@destinationElement"
unexpectedly found "{{#if this.sho" when slicing source, but expected "@renderInPlace"
WARNING: Detected collisions between .js and .ts files of the same name. This can result in nondeterministic build output; see https://git.io/JvIwo for more information.
  - vault/services/flash-messages.{js,ts}
⠋ building... [Package /assets/vendor.js]Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(str-length($svg), $slice) or calc(str-length($svg) / $slice)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
17 │     $loops: ceil(str-length($svg) / $slice);
   │                  ^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    /tmp/broccoli-1815072jdTS03ql6VUQ/out-1044-funnel_funnel_styles/app/styles/sass-svg-uri/_svg-uri.scss 17:18    svg-uri()
    /tmp/broccoli-1815072jdTS03ql6VUQ/out-1044-funnel_funnel_styles/app/styles/core/checkbox-and-radio.scss 75:12  @import
    /tmp/broccoli-1815072jdTS03ql6VUQ/out-1044-funnel_funnel_styles/app/styles/core.scss 25:9                      @import
    /tmp/broccoli-1815072jdTS03ql6VUQ/out-1044-funnel_funnel_styles/app/styles/app.scss 10:9                       root stylesheet

⠧ building... [ember-auto-import-webpack]autoprefixer: /home/cipherboy/GitHub/cipherboy/openbao/ui/assets/vault.css:10093:3: Replace fill-available to stretch, because spec had been changed
cleaning up...
Built project successfully. Stored in "../http/web_ui".
(node:1815072) [DEP0128] DeprecationWarning: Invalid 'main' field in '/home/cipherboy/GitHub/cipherboy/openbao/ui/node_modules/fireworm/package.json' of 'fireworm.js'. Please either fix that or report it to the module author
(Use `node-20 --trace-deprecation ...` to show where the warning was created)
2024-03-07T11:11:31.470-0500 [INFO]  core: enabled credential backend: path=userpass-1-a9b36ccb-70de-41f0-b0cf-db2a59105b8b/ type=userpass version=""
2024-03-07T11:11:31.541-0500 [INFO]  core: enabled credential backend: path=userpass-2-a9b36ccb-70de-41f0-b0cf-db2a59105b8b/ type=userpass version=""
2024-03-07T11:11:31.756-0500 [INFO]  core: disabled credential backend: path=auth/userpass-1-a9b36ccb-70de-41f0-b0cf-db2a59105b8b/
2024-03-07T11:11:31.770-0500 [INFO]  core: disabled credential backend: path=auth/userpass-2-a9b36ccb-70de-41f0-b0cf-db2a59105b8b/
2024-03-07T11:11:32.431-0500 [INFO]  core: enabled credential backend: path=userpass-1.3625532739781103/ type=userpass version=""
2024-03-07T11:11:32.462-0500 [INFO]  core: mount tuning of description successful: path=auth/userpass-1.3625532739781103/ description=""
2024-03-07T11:11:32.462-0500 [INFO]  core: mount tuning of token_type successful: path=auth/userpass-1.3625532739781103/ token_type=default-service
2024-03-07T11:11:32.906-0500 [INFO]  core: enabled credential backend: path=userpass-2.3625532739781105/ type=userpass version=""
2024-03-07T11:11:32.930-0500 [INFO]  core: mount tuning of description successful: path=auth/userpass-2.3625532739781105/ description=""
2024-03-07T11:11:32.930-0500 [INFO]  core: mount tuning of token_type successful: path=auth/userpass-2.3625532739781105/ token_type=default-service
2024-03-07T11:11:33.802-0500 [INFO]  core: enabled credential backend: path=auth-list-userpass-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/ type=userpass version=""
2024-03-07T11:11:34.048-0500 [INFO]  core: disabled credential backend: path=auth/auth-list-userpass-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/
2024-03-07T11:11:34.211-0500 [INFO]  core: enabled credential backend: path=auth-list-ldap-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/ type=ldap version=""
2024-03-07T11:11:34.478-0500 [INFO]  core: disabled credential backend: path=auth/auth-list-ldap-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/
2024-03-07T11:11:34.640-0500 [INFO]  core: enabled credential backend: path=auth-list-jwt-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/ type=jwt version=""
2024-03-07T11:11:35.037-0500 [INFO]  core: enabled credential backend: path=auth-list-oidc-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/ type=oidc version=""
2024-03-07T11:11:35.437-0500 [INFO]  core: enabled credential backend: path=auth-list-radius-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/ type=radius version=""
2024-03-07T11:11:35.719-0500 [INFO]  core: disabled credential backend: path=auth/auth-list-radius-a41e75ac-e8c4-4348-9114-e4a7a6431bb2/
2024-03-07T11:11:41.260-0500 [INFO]  core: enabled credential backend: path=cluster-userpass-ab50cd64-3c47-4a85-a815-c5e63f8e0c45/ type=userpass version=""
2024-03-07T11:11:43.165-0500 [INFO]  core: successful mount: namespace="" path=console-route-1/ type=kv version=""
2024-03-07T11:11:43.192-0500 [INFO]  core: successful mount: namespace="" path=console-route-2/ type=kv version=""
2024-03-07T11:11:43.217-0500 [INFO]  core: successful mount: namespace="" path=console-route-3/ type=kv version=""
2024-03-07T11:11:43.332-0500 [INFO]  core: successfully unmounted: path=console-route-1/ namespace=""
2024-03-07T11:11:43.359-0500 [INFO]  core: successfully unmounted: path=console-route-2/ namespace=""
2024-03-07T11:11:43.386-0500 [INFO]  core: successfully unmounted: path=console-route-3/ namespace=""

1..72
# tests 72
# pass  64
# skip  7
# todo  0
# fail  1
Testem finished with non-zero exit code. Tests failed.

Error: Command failed with exit code 1: ember test -f=!enterprise
    at makeError (/home/cipherboy/GitHub/cipherboy/openbao/ui/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/cipherboy/GitHub/cipherboy/openbao/ui/node_modules/execa/index.js:114:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /home/cipherboy/GitHub/cipherboy/openbao/ui/scripts/start-vault.js:73:7 {
  shortMessage: 'Command failed with exit code 1: ember test -f=!enterprise',
  command: 'ember test -f=!enterprise',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
make: *** [Makefile:11: test] Error 1

```

However, this still leaves much unfixed. It seems like perhaps additional updates failed to play nicely with the tests.

### PR DESCRIPTION
**Pull request by JanMa**
This fixes the remaining failing UI Tests and enabled the test suite again.

Resolves #154

## Target Release

2.2.0

### CODE REVIEW

File: ui/tests/acceptance/pki/pki-action-forms-test.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ module('Acceptance | pki action forms test', function (hooks) {
       assert.dom(S.configuration.emptyState).doesNotExist();
       // The URLs section is populated based on params returned from OpenAPI. This test will break when
       // the backend adds fields. We should update the count accordingly.
-      assert.dom(S.configuration.urlField).exists({ count: 4 });
+      assert.dom(S.configuration.urlField).exists({ count: 5 });
```
Comment (cipherboy) [inline ui/tests/acceptance/pki/pki-action-forms-test.js:187]: Does anything else need to change as a result of this?

File: ui/tests/acceptance/settings/auth/configure/section-test.js [hunk 2]
```diff
@@ -60,14 +60,14 @@ module('Acceptance | settings/auth/configure/section', function (hooks) {
...
+  for (const type of ['ldap', 'kubernetes']) {
     test(`it shows tabs for auth method: ${type}`, async function (assert) {
       const path = `${type}-showtab-${this.uid}`;
       await cli.consoleInput(`write sys/auth/${path} type=${type}`);
       await cli.enter();
       await indexPage.visit({ path });
-      // aws has 4 tabs, the others will have 'Configuration' and 'Method Options' tabs
-      const numTabs = type === 'aws' ? 4 : 2;
+      // ldap has 3 tabs, the others will have 'Configuration' and 'Method Options' tabs
+      const numTabs = 2;
```
Comment (cipherboy) [inline ui/tests/acceptance/settings/auth/configure/section-test.js:70]: Comment and code don't match?

**Review by cipherboy — APPROVED** [review]
@JanMa Two questions, but I don't think they need to hold up merge given the tests pass. Thank you!!

File: ui/tests/acceptance/pki/pki-action-forms-test.js (hunk 1 above)
Comment (JanMa) [reply ui/tests/acceptance/pki/pki-action-forms-test.js:187]: I don't think so

**Pull request merged**